pkg/kubelet/cm/dra/plugin/dra_plugin.go

```diff
@@ -143,17 +159,21 @@ func (p *DRAPlugin) NodePrepareResources(
 	ctx = klog.NewContext(ctx, logger)
 	logger.V(4).Info("Calling NodePrepareResources rpc", "request", req)
 
+	conn, err := p.getOrCreateGRPCConn()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get gRPC connection: %w", err)
+	}
+
```

Comment:
NodePrepareResources doesn't belong to health monitoring API and its connection is handled differently (see https://github.com/kubernetes/kubernetes/pull/132058 for details).

Instead of making DRA APIs to use not monitored connection, it would be good to make a health monitoring to use monitored one, I believe. However, this might require some refactoring work.